src/main/webapp/app/exam/manage/test-runs/test-run-management.component.html

```diff
@@ -75,23 +75,23 @@ <h2><span jhiTranslate="artemisApp.examManagement.testRun.title">Test Run Manage
                 </tr>
             </thead>
             <tbody>
-                <tr *ngFor="let testRun of testRuns; trackBy: trackId">
-                    <td class="d-none d-md-table-cell">
+                <tr *ngFor="let testRun of testRuns; trackBy: trackId" [id]="'testrun-' + testRun.id">
+                    <td class="testrun-id d-none d-md-table-cell">
                         <a [routerLink]="[testRun.id]">{{ testRun.id }}</a>
                     </td>
                     <td>
                         <a> {{ testRun.user?.name || '--' }}</a>
                     </td>
-                    <td>{{ testRun.workingTime || 0 | artemisDurationFromSeconds }}</td>
-                    <td>{{ (testRun.started ? 'global.generic.yes' : 'global.generic.no') | artemisTranslate }}</td>
-                    <td>{{ (testRun.submitted ? 'global.generic.yes' : 'global.generic.no') | artemisTranslate }}</td>
+                    <td class="working-time">{{ testRun.workingTime || 0 | artemisDurationFromSeconds }}</td>
```

Comment:
why do you use css classes here? I thought we want to consistently use ids?

Reply:
Since ids should only be used once per document, I decided to use classes here. But the table rows have a id attribute `testrun-{id}`, so within cypress I reference the elements with `#testrun-{id} .working-time`. A alternative could be to use ids like `testrun-{id}-working-time`. Personally I prefer the first approach, but I'm open for feedback!